nudge/scripts/deploy-elasticbeanstalk-docker.sh

```diff
@@ -0,0 +1,50 @@
+application=$1
+environment=$2
+db_username=$3
+db_password=$4
+db_name=$5
+db_hostname=$6
+is_forked_pr=$7
+build_id=$8
+web3_provider_host=${9}
+graphql_base_url=${10}
+
+function check_is_forked_pr()
+{
+    if [ is_forked_pr = "true" ]; then
```

Comment:
Can you make an issue for this? I think it bears investigating further, though it doesn't block this PR